src/layer/dataset.ts

```diff
@@ -32,6 +36,8 @@ export const DATASET_EOCLOUD_S1GRD: Dataset = {
   searchIndexUrl: 'https://eocloud.sentinel-hub.com/index/s1/v1/search',
   findDatesUTCUrl: 'https://eocloud.sentinel-hub.com/index/s1/v1/finddates',
   orbitTimeMinutes: 49.3,
+  minDate: new Date(Date.UTC(2014, 4 - 1, 3, 0, 0, 0)),
```

Comment:
Talked live, the data on EOCloud is available since October 2014, not april ([source](https://docs.sentinel-hub.com/api/latest/#/data/Sentinel-1-GRD?id=endpoint-locations))

Reply:
Thank you, fixed!

Reply:
👍  Resolving